src/Common/src/System/Xml/Base64Encoder.cs

```diff
@@ -5,9 +5,9 @@ namespace System.Xml
 {
     internal abstract partial class Base64Encoder
     {
-        byte[] leftOverBytes;
-        int leftOverBytesCount;
-        char[] charsLine;
+        private byte[] leftOverBytes;
```

Comment:
Adding explicit private modifiers is good, but is there a reason some of these fields didn't / don't follow the leading-underscore naming convention? I'd prefer to fix that all up at the same time unless there was a reason these didn't follow that convention in the first place.